src/index.js

```diff
@@ -77,11 +77,10 @@ function openRegisterDialog() {
             password: '',
             activation_keys: '',
             org: '',
-            insights: true,
+            insights: subscriptionsClient.insightsAvailable,
```

Comment:
this determines whether to register to insights by default; it should be still true even when insights-client is not available (which can be fully determined only after trying to install it later on)

hence, this should be `true` again